app/src/main/res/layout/fragment_shop_update.xml

```diff
@@ -164,6 +163,94 @@
+                        app:imageUrl="@{viewModel.imageAvatar}"
+                        tools:ignore="ContentDescription,UnusedAttribute"
+                        />
+                    <ImageButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_centerVertical="true"
+                        android:layout_gravity="center"
+                        android:layout_marginLeft="@dimen/dp_20"
+                        android:layout_marginStart="@dimen/dp_20"
+                        android:layout_toEndOf="@+id/image_avatar"
+                        android:layout_toRightOf="@+id/image_avatar"
+                        android:contentDescription="@string/avatar"
+                        android:onClick="@{()-> viewModel.onClickChooseAvatar()}"
+                        app:srcCompat="@drawable/ic_add_image"
+                        />
+                </RelativeLayout>
+                <RelativeLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="@dimen/dp_20"
+                    android:gravity="center"
+                    >
+                    <ImageView
+                        android:id="@+id/image_cover"
+                        android:layout_width="@dimen/dp_250"
+                        android:layout_height="@dimen/dp_150"
+                        android:layout_marginLeft="@dimen/dp_10"
+                        android:layout_marginStart="@dimen/dp_10"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:background="@drawable/border_layout"
+                        android:backgroundTint="@color/color_gray_white"
+                        android:contentDescription="@string/cover_image"
+                        android:padding="@dimen/dp_5"
+                        android:paddingEnd="@dimen/dp_5"
+                        android:paddingTop="@dimen/dp_5"
+                        android:scaleType="centerCrop"
+                        app:imageUrl="@{viewModel.imageCover}"
+                        tools:ignore="UnusedAttribute"
+                        />
+                    <ImageButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_centerVertical="true"
+                        android:layout_gravity="center_vertical"
+                        android:layout_marginLeft="@dimen/dp_10"
+                        android:layout_marginStart="@dimen/dp_10"
+                        android:layout_toEndOf="@+id/image_cover"
+                        android:layout_toRightOf="@+id/image_cover"
+                        android:onClick="@{()-> viewModel.onClickChooseCoverImage()}"
+                        android:src="@drawable/ic_add_image"
```

Comment:
ERROR: [VectorDrawableCompat] When using VectorDrawableCompat, you need to use `app:srcCompat`.
 To use VectorDrawableCompat, you need to make two modifications to your project. First, set `android.defaultConfig.vectorDrawables.useSupportLibrary = true` in your `build.gradle` file, and second, use `app:srcCompat` instead of `android:src` to refer to vector drawables.